resources/run_adf.sh

```diff
@@ -122,6 +126,19 @@ check_version() {
   fi
 }
 
+check_compute_node() {
```

Comment:
Just to double check, now ADF uses a completely separate mongod as well as the mongod where the collections are stored?

Reply:
This is correct.